src/app/app.ts

```diff
@@ -183,6 +184,50 @@ export function view(state: IState, actions: any) {
             actions.graph.deleteNode,
           )
         : Empty(),
+
+        html.div(
+          {
+            id: 'drag',
+            style: {
+              position: 'absolute',
+              bottom: 0,
+              right: 0,
+              height: '200px',
+              width: '200px',
+              background: 'White',
+            },
+            ondragover: (ev: Event) => {
+              ev.preventDefault()
+              ev.stopPropagation()
+              logger.log('dragging')
+            },
+            ondragleave: (ev: Event) => {
+              ev.preventDefault()
+              ev.stopPropagation()
+              logger.log('leaving')
+            },
+            ondrop: (ev: any) => {
+              ev.preventDefault()
+              ev.stopPropagation()
+              for (const f of ev.dataTransfer.files) {
+                logger.log(f.path)
```

Comment:
change this to a `debug` instead of a `log`. And change the message to "Dropped file path = f.path"